compiler/one-cmds/one_build.py

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env python
+
+# Copyright (c) 2020 Samsung Electronics Co., Ltd. All Rights Reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import configparser
+import os
+import subprocess
+import sys
+
+import one_utils as _utils
+
+
+def _get_parser():
+    parser = argparse.ArgumentParser(
+        description='command line tool to run ONE drivers in customized order')
+
+    # version
+    parser.add_argument(
+        '-v',
+        '--version',
+        action='store_true',
+        help='show program\'s version number and exit')
+
+    # configuration file
+    config_group = parser.add_argument_group('configuration arguments')
+    config_group.add_argument(
+        '-C', '--config', type=str, help='run with configuation file')
+    config_group.add_argument(
+        '-S',
+        '--section',
```

Comment:
Do we really need this to get from the user?

Reply:
@seanshpark There are two options for users. If `--section` option is given, just run the section. If option is not given, all section in config file will be run sequentially.

Reply:
> If option is not given, all section in config file will be run sequentially.

This means, config file is not just provides static information but a control information from `run sequentially.`.
I don't this is a good idea. It would be better not to include this implict behaviour.

Reply:
In another point, users may want to get controllability of execution sequence somehow . Let's think about this...